autopts/ptsprojects/zephyr/gap.py

```diff
@@ -59,6 +59,7 @@ class CHAR:
 iut_svc_data = '1111'
 iut_flags = '11'
 iut_svcs = '1111'
+iut_uri = UriScheme.https + 'github.com/intel/auto-pts'.encode()
```

Comment:
```suggestion
iut_uri = UriScheme.https + 'github.com/auto-pts/auto-pts'.encode()
```

Reply:
yeah, I'd have to update ixit with encoded value,  will do that in separate PR

Reply:
I've updated anyway :)